andela/travel_tool_front
# #161044635 View travel checklist for 'Open' requests

#### What does this PR do?
This PR displays the travel checklists for open requests

#### Description of Task to be completed?
- Add the `travel checklist` button to `request table menu`
- Display travel checklist on a modal when `travel checklist button` is clicked

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login and make a `new request`
- Click on the `new request` ellipsis and then click on `travel checklist` on the modal dropdown
- The travel checklist modal should pop up and show the checklist for the travel request.

#### Any background context you want to provide?
- You need to clone the [backend](https://github.com/andela/travel_tool_back.git) to test this functionality.


#### What are the relevant pivotal tracker stories?
[#161044635](https://www.pivotaltracker.com/story/show/161044635)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-10-13 at 6 22 22 pm" src="https://user-images.githubusercontent.com/27569442/46908079-09c57b00-cf15-11e8-97dd-f7b8e7bc1a30.png">

<img width="1433" alt="screen shot 2018-10-13 at 6 23 23 pm" src="https://user-images.githubusercontent.com/27569442/46908096-6163e680-cf15-11e8-85b2-92842b5e174b.png">

#### Questions:
N/A
